public/docs/ts/latest/guide/index.jade

```diff
@@ -1,27 +1,19 @@
 block includes
   include ../_util-fns
 
-figure
-  img(src="/resources/images/devguide/intro/people.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px")
-
+- var langName = current.path[1] == 'ts' ? 'TypeScript' : 'JavaScript'
 :marked
-  This is a practical guide to Angular for experienced programmers who
-  are building client applications in HTML and #{_Lang}. <br class="l-clear-left">
+  This page describes the Angular documentation at a high level.
+  If you're new to Angular, you may want to visit "[Learning Angular](learning-angular.html)" first.
 
-  ## Organization
+  ## Themes
 
   The documentation is divided into major thematic sections, each
   a collection of pages devoted to that theme.
+  <br clear="all">
```

Comment:
If you have deleted the `people.png` figure, you can drop this `<br clear...>` line.